### ISSUE TITLE
web-fragment.xml in krazo-core is preventing <distributable/> in web.xml

### ISSUE THREAD

**Issue by jcputney** [eclipse-ee4j/krazo#265]
When using version 1.1.0 in my WAR, I cannot set `<distributable/>` in my web.xml on Wildfly and I receive the following message in my logs:

```
WFLYUT0081: The deployment web.war will not be distributable because this feature is disabled in web-fragment.xml of the module krazo-core-1.1.0.jar
```

Is it possible to override this somehow? Or possible to add `<distributable/>` to web-fragment.xml? 

If so, a 1.1.1 release would be great, for those of us that can't move to Jakarta EE 9 yet.

**Comment by chkal** [comment]
Thanks a lot for reporting this. I just created #266 to address this.

I agree that this fix should go into the 1.1.x maintenance branch as well. I'll create a corresponding PR as soon as #266 is merged.

Regarding a workaround: Maybe it could help to exclude Krazo's `web-fragment.xml`? AFAIK this can be done by including a `<absolute-ordering>` element in your `web.xml` and listing all the fragments that you actually want to be included. But I'm not sure if this will really work. But perhaps it is worth a try.

**Comment by chkal** [comment]
Sorry for the delayed reply. I'm currently in the process of moving to a new apartment, and therefore I don't find much time to watch GitHub issues. I'll bring this up during our next team meeting.

**Comment by jcputney** [comment]
@chkal the workaround listed above does not work. Is there any movement on a 1.1.1 release?

**Comment by chkal** [comment]
I just started a discussion about this on our [mailing list](https://accounts.eclipse.org/mailing-list/krazo-dev). I already cherry-picked the fix to the `release-1.1.0` branch.

**Comment by chkal** [comment]
We just released version 1.1.1 of Krazo which includes this fix. The release should show up in Maven central in the next hours.

**Comment by jcputney** [comment]
@chkal thanks!
